tests/Doctrine/Migrations/Tests/VersionTest.php

```diff
@@ -599,6 +605,48 @@ public function testRunWithInsertNullValue() : void
         self::assertContains('with parameters ([])', $messages[1]);
     }
 
+    /**
+     * @dataProvider getExecutedAtTimeZones
+     */
+    public function testExecutedAtTimeZone(string $timeZone) : void
+    {
+        $originalTimeZone = date_default_timezone_get();
+
+        date_default_timezone_set($timeZone);
```

Comment:
I'd put whole test after this statement inside `try { ... }` and then `date_default_timezone_set($originalTimeZone);` inside `finally`, otherwise when the test fails, the timezone will stay messed up.

Reply:
FIWI, there's `TestCase::iniSet()` which could be use to simplify the cleanup.

Reply:
Cool, didn't know it exists. 👍